frontend/lib/norent/letter-content.tsx

```diff
@@ -46,6 +46,14 @@ const FullName: React.FC<NorentLetterContentProps> = (props) => (
   </>
 );
 
+export const getStreetWithApt = ({
+  street,
+  aptNumber,
+}: Pick<NorentLetterContentProps, "street" | "aptNumber">) => {
+  if (!aptNumber) return street;
+  return `${street} #${aptNumber}`;
+};
+
```

Comment:
Nice! Is there a reason that you implemented this as this if statement rather than just with a simple ternary operator? Just curious!

Reply:
Oh hehe, not really, I just ended up writing it that way!  Although I generally do find `if` statements more readable than ternaries (in fact Python famously refused to support them for a really long time b/c of readability concerns, though it did eventually adopt the `boop = 1 if True else 2` style syntax, which I think is kinda the best of both worlds...